vizro-core/docs/pages/user-guides/aggrid.md

```diff
@@ -0,0 +1,281 @@
+# How to use AGGrids
+
+This guide shows you how to use the [AG Grid](https://www.ag-grid.com/) to visualize your tabular data in the dashboard.
```

Comment:
"the AG Grid" or "an AG Grid" ?

Also, I wasn't sure what one of these was and followed the trail right back to the original ag-grid website, which still doesn't explain what AG stands for. So I think it's fine that we don't either.

Reply:
I usually skip the article completely.  So "This guide shows you how to use AG Grid....."

For background, you can find more about what the AG stands for here:  https://blog.ag-grid.com/reasons-to-choose-ag-grid-as-your-javascript-datagrid/
But the AG (agnostic) no longer applies when you use Dash AG Grid